src/styles/blocks/month.scss

```
@@ -0,0 +1,52 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+body {
+  padding: 0;
+  margin: 50px 0 0;
+  @extend %centre-flex;
+}
+
+.month {
+  display: flex;
+  align-content: flex-start;
+  flex-wrap: wrap;
+  gap: #{$card-gapNborder};
+  width: 750px;
+
+  &__day {
+    @extend %centre-flex;
+    @extend %card-perimeter;
+    background-color: #eee;
+    border: #{$card-gapNborder} solid #000;
+    transition-duration: 0.5s;
+
+    &:hover {
+      background-color: #FFC0CB;
+      cursor: pointer;
+      transform: translateY(-20px);
+    }
+  }
+}
+
+@for $index from 1 through 31{
+  .month__day:nth-child(#{$index}):before {
+    content: '#{$index}';
+    color: #000;
+    font-size: 30px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+}
+
+@each $num, $day in $week-days {
+  .start-#{$day} {
+    margin-left: $num * ($card-square + ($card-gapNborder * 3)) ;
```

I don't think you need multiplying it by `3`
```suggestion
    margin-left: $num * ($cell-width + $cell-gap);
```